ci/Jenkinsfiles/release.groovy

```diff
@@ -79,10 +79,21 @@ pipeline {
     DOCKER_NAMESPACE = 'nuxeo'
     NUXEO_IMAGE_NAME = 'nuxeo'
     SLIM_IMAGE_NAME = 'slim'
+    SLACK_CHANNEL = 'platform-notifs'
   }
 
   stages {
 
+    stage('Notify Promotion start on slack') {
+      steps {
+        script {
+          if (env.DRY_RUN != "true") {
+            slackSend(channel: "${SLACK_CHANNEL}", color: "#0167FF", message: "Promoting ${BRANCH_NAME} from ${CURRENT_VERSION} to ${RELEASE_VERSION}: ${BUILD_URL}")
```

Comment:
I'd go with a message similar to the build description we set at the end in case of success:  "Starting release ${RELEASE_VERSION} from build ${params.BUILD_VERSION}: : ${BUILD_URL}", which will translate to "Starting release 11.3 from build 11.3.47: https://xxx" for instance.